### PR TITLE
Types of tokens

### PR DESCRIPTION
**Pull request by fadeev**
Added a "Types of Tokens" section:

Screenshot 2024-02-08 at 12 52 53

Added an inline table with ZETA token addresses on connected chains:

Screenshot 2024-02-08 at 12 53 59

Added a summary to the ZRC-20 section:

> Native gas tokens of connected blockchains and whitelisted ERC-20 tokens can be deposited to ZetaChain as ZRC-20 tokens. During the deposit process, the native/ERC-20 tokens are transferred to and locked in the TSS address/ERC-20 custody contract and ZRC-20 tokens are minted on ZetaChain and deposited to the recipient address.

> ZRC-20 tokens can be withdrawn from ZetaChain to connected blockchains. During the withdrawal process, ZRC-20 tokens are burnt on ZetaChain and native/ERC-20 tokens are transferred to the recipient address on the connected chain from a TSS address/ERC-20 custody contract.

> ZRC-20 tokens can only be minted by the ZetaChain protocol. An ERC-20 token deployed on ZetaChain does not have the properties of ZRC-20 and can't be withdrawn from ZetaChain to a connected chain.

> The "same" ERC-20 token from two connected blockchains is represented as two different ZRC-20 tokens on ZetaChain. For example, USDT from Ethereum is represented as ZRC-20 USDT from Ethereum, and USDT from BSC is represented as ZRC-20 USDT from BSC. They are not considered the same asset by ZetaChain, but they can be swapped. That's how the transfer of the "same" ERC-20 asset can be implemented on ZetaChain: by depositing an ERC-20 (chain A), swapping this ZRC-20 (chain A) to an ZRC-20 (chain B), and withdrawing the ZRC-20 (chain B) to chain B as ERC-20.

### CODE REVIEW

File: docs/developers/tokens/zeta.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  ]
+}
+```
+
+To convert the value amount from `azeta` to ZETA, divide it by 10¹⁸. In the
+example above the balance is 10 ZETA.
+
+## Wrapped ZETA on ZetaChain
+
+ZETA can exists on ZetaChain in a wrapped form (WZETA) as a
```
Comment (bbbeeeee) [inline docs/developers/tokens/zeta.mdx:37]: *can exist

File: docs/developers/tokens/zeta.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+[WETH9 token](https://github.com/zeta-chain/protocol-contracts/blob/main/contracts/zevm/WZETA.sol).
+WZETA is primarily used in internal liquidity pools on ZetaChain paired with
+native gas tokens of connected blockchains (for example, gETH/WZETA pair).
```
Comment (bbbeeeee) [inline docs/developers/tokens/zeta.mdx:40]: Maybe we could add something like:

WZETA may be used or required by other dApps that are built to interface with ERC-20-like tokens.

File: docs/developers/tokens/zeta.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+To wrap native ZETA and turn it into WZETA, send it to or use the `deposit`
+method of the `zetaToken` [contract on ZetaChain](/reference/contracts).
+
+## Wrapped ZETA on Connected Blockchains
```
Comment (bbbeeeee) [inline docs/developers/tokens/zeta.mdx:45]: I think it should be called just "ZETA on Connected Blockchains"

File: docs/developers/tokens/erc20.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+# ERC-20 on ZetaChain
+
+ZetaChain is fully compatible with the Ethereum Virtual Machine (EVM) and
+[ERC-20](https://ethereum.org/en/developers/docs/standards/tokens/erc-20) token
+contracts can be deployed to ZetaChain permissionlessly.
+
+:::note
+
+An ERC-20 token deployed to ZetaChain **does not** have the properties of
+ZRC-20, specifically, it can't be withdrawn from ZetaChain to a connected chain.
```
Comment (bbbeeeee) [inline docs/developers/tokens/erc20.md:14]: link to ZRC-20?

File: docs/developers/tokens/zrc20.md [hunk 5]
```diff
@@ -18,9 +15,35 @@ place extremely simple — as if they were all on a single chain.
...
+
+The "same" ERC-20 token from two connected blockchains is represented as two
+different ZRC-20 tokens on ZetaChain. For example, USDT from Ethereum is
+represented as ZRC-20 USDT from Ethereum, and USDT from BSC is represented as
+ZRC-20 USDT from BSC. They are not considered the same asset by ZetaChain, but
+they can be swapped. That's how the transfer of the "same" ERC-20 asset can be
+implemented on ZetaChain: by depositing an ERC-20 (chain A), swapping this
+ZRC-20 (chain A) to an ZRC-20 (chain B), and withdrawing the ZRC-20 (chain B) to
+chain B as ERC-20.
+
```
Comment (bbbeeeee) [inline docs/developers/tokens/zrc20.md:43]: Maybe we could add a naming convention section?

File: docs/developers/tokens/zeta.mdx (hunk 1 above)
Comment (bbbeeeee) [reply docs/developers/tokens/zeta.mdx:37]: Also, I think it's worth explaining what wrapped means in this case.

**Review by bbbeeeee — COMMENTED** [review]
Looking great - left a few comments!

File: docs/developers/tokens/zeta.mdx (hunk 1 above)
Comment (fadeev) [reply docs/developers/tokens/zeta.mdx]: 183c42b 68af21c

Comment (fadeev) [reply docs/developers/tokens/zeta.mdx]: I do need a good spellcheck for VS Code. You'd think with Copilot that would be part of the package 😁

File: docs/developers/tokens/zeta.mdx (hunk 2 above)
Comment (fadeev) [reply docs/developers/tokens/zeta.mdx]: 6e70f1f

Comment (fadeev) [reply docs/developers/tokens/zeta.mdx]: 6e70f1f

File: docs/developers/tokens/zeta.mdx (hunk 3 above)
Comment (fadeev) [reply docs/developers/tokens/zeta.mdx]: 2185fc6

File: docs/developers/tokens/erc20.md (hunk 4 above)
Comment (fadeev) [reply docs/developers/tokens/erc20.md]: 32e7187

File: docs/developers/tokens/zrc20.md (hunk 5 above)
Comment (fadeev) [reply docs/developers/tokens/zrc20.md:43]: I will definitely add a naming convention section as soon as we make a definitive decision on naming:

https://github.com/zeta-chain/node/issues/1734

**Review by bbbeeeee — APPROVED** [review]

**Pull request merged**